### PR TITLE
Chinese Language Create zh_cn.json

### PR DESCRIPTION
**Pull request by GodGun968**

### CODE REVIEW

File: common/src/main/resources/assets/rrls/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "text.autoconfig.rrls.title": "移除重新加载屏幕(RemoveReloadingScreen)",
+  "text.autoconfig.rrls.option.enabled": "隐藏重新加载屏幕",
+  "text.autoconfig.rrls.option.rgbProgress": "彩色进度条",
+  "text.autoconfig.rrls.option.rgbProgress.@Tooltip": "§c可能引起癫痫！",
```
Comment (dima-dencep) [inline common/src/main/resources/assets/rrls/lang/zh_cn.json:5]: §c ?

File: common/src/main/resources/assets/rrls/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "text.autoconfig.rrls.title": "移除重新加载屏幕(RemoveReloadingScreen)",
```
Comment (dima-dencep) [inline common/src/main/resources/assets/rrls/lang/zh_cn.json:2]: Leave either the translated text or the English

**Review by dima-dencep — CHANGES_REQUESTED** [review]

File: common/src/main/resources/assets/rrls/lang/zh_cn.json (hunk 1 above)
Comment (GodGun968) [reply common/src/main/resources/assets/rrls/lang/zh_cn.json:5]: I want it to stand out because epilepsy is no joke

Comment (dima-dencep) [reply common/src/main/resources/assets/rrls/lang/zh_cn.json:5]: Okay, I'll think about highlighting it everywhere.

File: common/src/main/resources/assets/rrls/lang/zh_cn.json (hunk 2 above)
Comment (GodGun968) [reply common/src/main/resources/assets/rrls/lang/zh_cn.json:2]: The original text is used to distinguish mod (on the mod website and when searching for mod in English ), the translated text is for explanatory purposes

**Pull request merged**